docs/en-US/component/switch.md

```diff
@@ -90,8 +90,8 @@ switch/prevent-switching
 | inline-prompt         | whether icon or text is displayed inside dot                                                                                                    | boolean                   | —               | false   |
 | active-icon           | component of the icon displayed when in `on` state, overrides `active-text`                                                                     | string / Component        | —               | —       |
 | inactive-icon         | component of the icon displayed when in `off` state, overrides `inactive-text`                                                                  | string / Component        | —               | —       |
-| active-text           | text displayed when in `on` state                                                                                                               | string                    | —               | —       |
-| inactive-text         | text displayed when in `off` state                                                                                                              | string                    | —               | —       |
+| active-text           | text displayed when in `on` state. only the first character will be displayed if `inline-prompt` is true                                        | string                    | —               | —       |
```

Comment:
This description would be more appropriate in inline-prompt

Reply:
Yes, I agree and appended a new commit to change description of inline-prompt